### PR TITLE
fix(documentation): discrepancies - opensourceguidelines

### LINKED ISSUE #271: Resolve discrepancies between languages
Lets resolve all discrepancies found in the previous comparison that are easy enough to solve , if the issues are more complex lets create new issues in the backlog

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### GH Issue

Resolves #271 
Related to spanish: https://github.com/eoscostarica/guias.eoscostarica.io/pull/223

### Steps to test
1. Check content
1.
1.

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/open-source-guidelines.md [hunk 1]
```diff
@@ -8,41 +8,40 @@ sidebar_label: Open Source Guidelines
 
 We use a Kanban-style board to prioritize the work. For example the [EOS rate project board](https://github.com/eoscostarica/eos-rate/projects/1).
 
-We have added a additional column to the default automated board in order to maintain a prioritized To Do column.
+We have added a additional column to the default automated board in order to maintain a prioritized `To` Do column.
```
Comment (xavier506) [inline docs/open-source-guidelines.md:11]: maintain a prioritized `To Do` column.

File: docs/open-source-projects.md [hunk 2]
```diff
@@ -79,9 +76,9 @@ We view the EOS Foundation as a container for facilitating code, capital, and co
 
 - **GitHub** : https://github.com/eoscostarica/eos-foundation
 
-![EOS Fundation](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Fundation_2.PNG)
+![EOS Fundation](https://raw.githubusercontent.com/eoscostarica/guias.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Fundation.PNG)
```
Comment (xavier506) [inline docs/open-source-projects.md:79]: guias?

File: docs/devops.md [hunk 3]
```diff
@@ -93,20 +88,21 @@ Terraform is an infrastructure software as code, developed by HashiCorp. It enab
...
-### Introduction to HashiCorp Terraform 
+## Introduction to HashiCorp Terraform 
 
 <figure class="video_container">
   <iframe width="100%" height="315" src="https://www.youtube.com/embed/h970ZBgKINg" frameborder="0" allowfullscreen="true">
   </iframe>
 </figure>
 
-## Introduction to Docker and Kubernetes 
+## Infraestructura como código en EOS CR
```
Comment (Edgar097) [inline docs/devops.md:98]: Why "Infraestructura como código en EOS CR"?

**Comment by Edgar097** [comment]
thanks for being so carefully aware of the details!

**Comment by xavier506** [comment]
Lets agree to the following text styles 

- [ ] Title Case for any Headers
- [ ] All sentences begin (including list items) with uppercase letter and ends with a period. (

File: docs/devops.md [hunk 4]
```diff
@@ -82,7 +77,7 @@ The process of managing and provisioning computer data centers through machine-r
 
 * * *
 
-### Introduction to Terraform
```
Comment (xavier506) [inline docs/devops.md:85]: H3

File: docs/devops.md [hunk 5]
```diff
@@ -93,20 +88,21 @@ Terraform is an infrastructure software as code, developed by HashiCorp. It enab
 
 * * *
 
-### Introduction to HashiCorp Terraform 
+## Introduction to HashiCorp Terraform
```
Comment (xavier506) [inline docs/devops.md:91]: H3

File: docs/devops.md (hunk 4 above)
Comment (xavier506) [reply docs/devops.md:85]: Introduction to HashiCorp Terraform

File: docs/devops.md (hunk 5 above)
Comment (xavier506) [inline docs/devops.md:91]: Remove header entirely

File: docs/devops.md (hunk 3 above)
Comment (leisterfrancisco) [reply docs/devops.md:98]: Thank you very much for your observation

File: docs/open-source-guidelines.md (hunk 1 above)
Comment (leisterfrancisco) [reply docs/open-source-guidelines.md:11]: My bad, thank you for your observation

File: docs/open-source-projects.md (hunk 2 above)
Comment (leisterfrancisco) [reply docs/open-source-projects.md:79]: you right, thank you very much

File: docs/open-source-projects.md [hunk 6]
```diff
@@ -79,10 +76,10 @@ We view the EOS Foundation as a container for facilitating code, capital, and co
 
 - **GitHub** : https://github.com/eoscostarica/eos-foundation
 
-![EOS Fundation](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Fundation_2.PNG)
-
 ![EOS Fundation](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Fundation.PNG)
```
Comment (xavier506) [inline docs/open-source-projects.md:79]: Foundation

File: docs/open-source-projects.md [hunk 7]
```diff
@@ -79,10 +76,10 @@ We view the EOS Foundation as a container for facilitating code, capital, and co
...
 
+![EOS Fundation](https://raw.githubusercontent.com/eoscostarica/guide.eoscostarica.io/master/static/img/OSS_screnshots/EOS_Fundation_2.PNG)
```
Comment (xavier506) [inline docs/open-source-projects.md:81]: Foundation

Comment (leisterfrancisco) [reply docs/open-source-projects.md]: Thank you for your observation 👍

File: docs/backend-boilerplate.md [hunk 8]
```diff
@@ -27,7 +28,7 @@ This template features all the latest tools and best practices in the industry.
 ### Step by step
 1. Clone the repo using the command `git clone --depth=1 https://github.com/eoscostarica/backend-boilerplate.git`
 1. Access the directory using the command: `cd backend-boilerplate`
-1. Copy `.env.example` file and then update the variables according to your needs.
+1. Copy `.env.example` file and then update the variables according to your needs
```
Comment (Edgar097) [inline docs/backend-boilerplate.md:31]: Maybe here instead of removing the dot, add it to the other sentences?

File: docs/devops.md [hunk 9]
```diff
@@ -34,11 +34,12 @@ In 2012, Puppet Labs “state of DevOps report” benchmarked 4,039 IT orgs usin
 
 ***
 
->“*The competitive advantage this capability creates is enormous, enabling faster feature time to market, increased customer satisfaction, market share, employee productivity, and happiness.”*   
-**-Phoenix Project**
+> “*The competitive advantage this capability creates is enormous, enabling faster feature time to market, increased customer satisfaction, market share, employee productivity, and happiness.”* ** - Phoenix Project **
 
 ***
 
+![](https://user-images.githubusercontent.com/5632966/105532539-95abdd00-5cb0-11eb-81ff-b854c4ec1784.png)
```
Comment (Edgar097) [inline docs/devops.md:41]: Around the web page we have images with and without description/name, i think is better to add a short name for the image.

File: docs/eos-learn/system-resources.md [hunk 10]
```diff
@@ -5,38 +5,44 @@ sidebar_label: System Resources
...
-## Network (NET)
+### Network (NET)
+
 As CPU and RAM, NET is also a very important resource in EOSIO-based blockchains. NET is the network bandwidth measured in bytes of transactions and it is referred to as `net bandwidth`.
 
-You can find more details about NET as a resource [here](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/net)
+You can find more details about NET as a resource [here](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/net).
 
-## Account resource limits
+## Account Resource imits
```
Comment (Edgar097) [inline docs/eos-learn/system-resources.md:39]: "Limits"

File: docs/tools/glossary.md [hunk 11]
```diff
@@ -53,7 +53,7 @@ The Airgrab model is designed to be a more capital efficient way to deploy dApp
 
 * * *
 
-### API
+### API (Application programming interface)
```
Comment (Edgar097) [inline docs/tools/glossary.md:56]: Think is better "Application Programming Interface" for this title

File: docs/tools/glossary.md [hunk 12]
```diff
@@ -124,7 +124,7 @@ It is the first DPOS blockchain equipped with IBC. Its goal is to refactor busin
 
 ### Bytecode canonical
```
Comment (Edgar097) [inline docs/tools/glossary.md:125]: Capitalization in Titles

File: docs/tutorials/github-keys.md [hunk 13]
```diff
@@ -6,25 +6,24 @@ sidebar_label: GitHub Keys
 
 ## Why we need GitHub Keys?
 
-Let's first define GitHub as a browser-based online control system of distributed version for those software developers using the Git revision control system. Git uses SSH public key authentication and not a password authentication.  Each user of the system has to generate a key. 
+Let's first define GitHub as a browser-based online control system of distributed version for those software developers using the Git revision control system. Git uses SSH public key authentication and not a password authentication.  Each user of the system has to generate a key.
 
 The `SSH` is also known as Secure Shell or Secure Socket Shell [More About SSH in GitHub](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/about-ssh). SSH keys are used to authenticate secure connections they are generate through a public key cryptographic algorithm, the most common being RSA or DSA. This is a one-way formula that ensures the public key can be derived from the private key but the private key cannot be derived from the public key.
 
-
 ## How to generate a new SSH key?
```
Comment (Edgar097) [inline docs/tutorials/github-keys.md:13]: here too

File: docs/backend-boilerplate.md (hunk 8 above)
Comment (leisterfrancisco) [reply docs/backend-boilerplate.md:31]: My bad, good observation.

File: docs/devops.md (hunk 9 above)
Comment (leisterfrancisco) [reply docs/devops.md:41]: Nice!! 🥇

File: docs/eos-learn/system-resources.md (hunk 10 above)
Comment (leisterfrancisco) [reply docs/eos-learn/system-resources.md:39]: I think it was extra finger, good observation.

File: docs/tools/glossary.md (hunk 11 above)
Comment (leisterfrancisco) [reply docs/tools/glossary.md:56]: Totally agree.

File: docs/tools/glossary.md (hunk 12 above)
Comment (leisterfrancisco) [reply docs/tools/glossary.md:125]: It tought I made this but it looks i didn't

File: docs/tutorials/github-keys.md (hunk 13 above)
Comment (leisterfrancisco) [reply docs/tutorials/github-keys.md:13]: mm not sure about this one, talking with Xavier, we agree that exist two cases to resolves this, first one is to remove the question mark and capitalize the sentences or well don't capitalize and use the question mark which what i did here

File: docs/devops.md (hunk 9 above)
Comment (leisterfrancisco) [reply docs/devops.md:41]: I'm going to remove this link, later in the same docs is use the same one in different context.

File: docs/eos-learn/consensus-mechanism.md [hunk 14]
```diff
@@ -39,27 +39,27 @@ Each producer receives a 12 block window to sign before the next producer starts
 
 If a producer node is not ready or unavailable, there is no one to produce the 12 blocks, so all speculative transactions are delayed until the next producer starts signing.
 
-### Byzantine Fault Tolerance
+### Byzantine fault tolerance
```
Comment (leisterfrancisco) [inline docs/eos-learn/consensus-mechanism.md:42]: Working on

File: docs/eos-learn/consensus-mechanism.md [hunk 15]
```diff
@@ -39,27 +39,27 @@ Each producer receives a 12 block window to sign before the next producer starts
...
 
 New blocks are considered reversible until they have been validated by 2/3 +1 of the active producers. This way if a producer node inserts an invalid block, the following nodes will reject it and the block will not be included unless 2/3 +1 of the producers group approve it.
 
->Example: A network of 21 active producers requires validation of 15 nodes (2/3 +1), which takes on average 90 seconds to obtain irreversibility of a new block.
+> **Example:** A network of 21 active producers requires validation of 15 nodes (2/3 +1), which takes on average 90 seconds to obtain irreversibility of a new block
 
-### Node Fault Tolerance
+### Node fault tolerance
```
Comment (leisterfrancisco) [inline docs/eos-learn/consensus-mechanism.md:48]: Working on

File: docs/eos-learn/eosio-protocol.md [hunk 16]
```diff
@@ -4,31 +4,32 @@ title: EOSIO Protocol
...
+- Designed for Sparse Header Light Client Validation
+- Scheduled Recurring Transactions
+- Time Delay Security
+- Designed for Parallel Execution of Context Free Validation Logic
+
+Another important feature of EOSIO based blockchain networks is it's flexible architecture, because many of the network's "rules of engagement" are defined through **system contracts**.
 
 This flexibility is achieved thanks to a set of smart contracts separated from the EOSIO core protocol. Because of this, the core functionality that validates blocks and transactions is separated from the **system contracts**.
 
 ## System contracts
```
Comment (leisterfrancisco) [inline docs/eos-learn/eosio-protocol.md:28]: Working on

File: docs/tools/eosio-sdk-libraries.md [hunk 17]
```diff
@@ -4,52 +4,53 @@ title: EOSIO SDK Libraries
 sidebar_label: EOSIO SDK Libraries
 ---
 
-APIs for integrating with EOSIO blockchains
+## SDK Libraries for EOSIO
```
Comment (xavier506) [inline docs/tools/eosio-sdk-libraries.md:7]: repetitive title?

Comment (leisterfrancisco) [reply docs/tools/eosio-sdk-libraries.md:7]: SDK and Library comes to be the same thing?

File: docs/engineering-culture.md [hunk 18]
```diff
@@ -8,7 +8,7 @@ We are a fast paced startup with really talented engineers who love open source
 
 Our values are reflected in the way we build software. 
 
-## At EOS Costa Rica we value:
+## At EOS Costa Rica We Value:
```
Comment (xavier506) [inline docs/engineering-culture.md:11]: At EOS Costa Rica we Value:

Comment (xavier506) [reply docs/engineering-culture.md:11]: no need to capitalize `we`

image

File: docs/devops.md [hunk 19]
```diff
@@ -47,7 +46,7 @@ DevOps principles are applied to and assisted by the following 3 Key areas all w
 - **Processes** – DevOps builds further upon Lean Agile, applying concepts of lean manufacturing to IT value streams.
 - **Tools** – DevOps relies on enhanced tools & technologies, including automation, telemetry, & collaboration tools.
 
-## What Is the Goal of DevOps?
+## What is the goal of DevOps?
```
Comment (xavier506) [inline docs/devops.md:49]: title case?

File: docs/devops.md [hunk 20]
```diff
@@ -23,7 +23,7 @@ It focuses on unifying Development & IT Operations through the principles of flo
 - **Deployment**: The deployment (or transport) of code to a higher-level environment, up to & including production.
 - **Release**: The defined scope and business objective for a set of features to be delivered in a certain timeline.
 
-## Why Implement DevOps?
+## Why implement DevOps?
```
Comment (xavier506) [inline docs/devops.md:26]: title case?

File: docs/devops.md (hunk 19 above)
Comment (leisterfrancisco) [reply docs/devops.md:49]: @xavier506 then remove the question mark, right?

File: docs/devops.md (hunk 20 above)
Comment (leisterfrancisco) [reply docs/devops.md:26]: Same here, @xavier506 then remove the question mark, right?

File: docs/tools/eosio-sdk-libraries.md (hunk 17 above)
Comment (leisterfrancisco) [reply docs/tools/eosio-sdk-libraries.md:7]: I change it to:
1. SDK for EOSIO
1. Libraries for EOSIO
1. SDK and Libraries for EOSIO

File: docs/engineering-culture.md (hunk 18 above)
Comment (leisterfrancisco) [reply docs/engineering-culture.md:11]: Solved

File: docs/tools/eosio-sdk-libraries.md (hunk 17 above)
Comment (xavier506) [reply docs/tools/eosio-sdk-libraries.md:7]: SDKs for EOSIO

**Comment by xavier506** [comment]
can you fix the conflicts and we can merge this PR

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by Edgar097 — APPROVED** [review]

**Pull request merged**